pkg/controller/operators/olm/operator.go

```diff
@@ -1255,3 +1321,51 @@ func (a *Operator) cleanupCSVDeployments(logger *logrus.Entry, csv *v1alpha1.Clu
 		}
 	}
 }
+
+func (a *Operator) ensureDeploymentAnnotations(logger *logrus.Entry, csv *v1alpha1.ClusterServiceVersion) error {
```

Comment:
Added this while looking into why copying was failing - small optimization that skips the walk through Failed if the annotations change in an operatorgroup.